checkov/terraform/checks/resource/aws/LambdaFunctionURLAuth.py

```diff
@@ -0,0 +1,21 @@
+from checkov.terraform.checks.resource.base_resource_negative_value_check import BaseResourceNegativeValueCheck
+from checkov.common.models.enums import CheckCategories
+
+
+class LambdaFunctionURLAuth(BaseResourceNegativeValueCheck):
+
+    def __init__(self):
+        name = "Ensure that lambda URL functions AuthType is not None"
+        id = "CKV_AWS_258"
+        supported_resources = ['aws_lambda_function_url']
+        categories = [CheckCategories.ENCRYPTION]
```

Comment:
```suggestion
        categories = [CheckCategories.GENERAL_SECURITY]
```
`ENCRYPTION` doesn't fit here, let's go with `GENERAL_SECURITY`